src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -0,0 +1,65 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.dto.ProductDto;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
Might as well add it later (or not at all because we're still a long way from the front-end) but consider adding `@CrossOrigin("*")`

Reply:
OK, I'll add